backend/src/controllers/complectations/complectations.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { getComplectationsById } from '@services/complectations/complectations.service';
+import httpStatus from 'http-status-codes';
+
+import type { TypedRequestQuery } from '@common/types/controller/controller';
+import type { NextFunction, Response } from 'express';
+
+type ModelIdInput = {
+  complectationId: string;
+};
+
+const getComplectations = async (
+  req: TypedRequestQuery<ModelIdInput>,
+  res: Response,
+  next: NextFunction,
+): Promise<void> => {
+  try {
+    const { complectationId } = req.query;
+    const ResponseDto = await getComplectationsById(complectationId);
+    res.json(ResponseDto).status(httpStatus.CREATED);
+  } catch (error) {
+    console.error(error);
+    if (error instanceof Error) {
+      res.sendStatus(404);
```

Comment:
httpStatus.NOT_FOUND

error instanceof Error 
What if we will have "connection refused to database?"  We will also get 404 but it is not true